dts/arm/st/h7/stm32h747.dtsi

```diff
@@ -5,3 +5,41 @@
  */
 
 #include <st/h7/stm32h7_dualcore.dtsi>
+
+/ {
+	/*
+	* The RAM memories placed here can be used by both cores M4/M7
```

Comment:
Aligned this with the H745 device tree which should have the same memory architecture. Anyone can confirm this is correct?

Reply:
Please have a check to https://github.com/zephyrproject-rtos/zephyr/pull/30530.
I'm reworking that part  a bit.

Reply:
> Aligned this with the H745 device tree which should have the same memory architecture. Anyone can confirm this is correct?

Yes. They have same memory architecture. They also have same ReferenceManual